cmd/get_logs.go

```diff
@@ -362,6 +417,16 @@ func (glc *getLogsCmd) getCloudName() string {
 	return ""
 }
 
+func updateLinuxNodesList(nodeList []v1.Node, agentPoolName string) []v1.Node {
+	var linuxNodeList []v1.Node
+	for _, node := range nodeList {
+		if strings.Split(node.Name, "-")[1] != agentPoolName {
```

Comment:
case insensitive comparison please

Reply:
I think the comparison here should not be case insensitive as the agentPool name from API model is used as part of the agent node name, But since the agentPool name only allows a-z0-9, there shouldn't be difference between case sensitive/insensitive. Updated.